conda/recipes/meta.yaml

```
@@ -0,0 +1,53 @@
+package:
+  name: nvjitlink
+  version: 0.1.0
+
+source:
+  #url: https://github.com/rapidsai/nvjitlink/archive/refs/tags/0.1.0.tar.gz
+  path: ../../
+
+build:
+  noarch: python
+  script:
+    # This is a workaround for library search path issues with the current
+    # cuda-nvcc package. They should be fixed by
+    # https://github.com/conda-forge/cuda-nvcc-feedstock/pull/24, at which
+    # point we can remove this.
+    - export CMAKE_PREFIX_PATH=${PREFIX}/targets/x86_64-linux/lib:$CMAKE_PREFIX_PATH
+    - pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake >=3.26.4
+    - sysroot_{{ target_platform }} 2.17
+    - cuda-version =12.0
+  host:
+    - libnvjitlink-dev
+    - libnvjitlink-static
+    - cuda-version =12.0
+    - python
+    - scikit-build-core
+  run:
+    - python
+    - numba >=0.57
+    - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
+test:
+  imports:
+    - nvjitlink
```

Did we want a smoke test like this with the extension module? Asking since it sounded like some errors got swallowed when `import`ing `nvjitlink`'s top-level package

```suggestion
    - nvjitlink._nvjitlink
    - nvjitlink
```

---

Just an update on this, still trying to figure out if this is actually a problem or something scrambled just for me locally.